content/applications/finance/accounting/payments.rst

```diff
@@ -30,114 +32,149 @@ At this point, the customer invoice or vendor bill is marked as :guilabel:`In pa
+   - If a payment is unreconciled, it still appears in the books but is no longer linked to the
      invoice.
-   - If you (un)reconcile a payment in a different currency, a journal entry is automatically
+   - If a payment is (un)reconciled in a different currency, a journal entry is automatically
      created to post the currency exchange gains/losses (reversal) amount.
-   - If you (un)reconcile a payment and an invoice having cash basis taxes, a journal entry is
-     automatically created to post the cash basis tax (reversal) amount.
+   - If a payment is (un)reconciled and an invoice with cash-basis taxes, a journal entry is
+     automatically created to post the cash-basis tax (reversal) amount.
 
 .. tip::
-   - If your main bank account is set as an :ref:`outstanding account
+   - If the main bank account is set as an :ref:`outstanding account
      <bank/outstanding-accounts>`, and the payment is registered in Odoo (not through a related
      bank statement), invoices and bills are directly registered as :guilabel:`Paid`.
 
+.. _accounting/payments/not-tied:
+
 Registering payments not tied to an invoice or bill
 ===================================================
 
-When a new payment is registered via :menuselection:`Customers / Vendors --> Payments` menu, it is
-not directly linked to an invoice or bill. Instead, the account receivable or the account payable is
-matched with the **outstanding account** until it is manually matched with its related invoice or
+When a new payment is registered via the :menuselection:`Customers / Vendors --> Payments` menu, it
+is not directly linked to an invoice or bill. Instead, the account receivable or the account payable
+is matched with the **outstanding account** until it is manually matched with its related invoice or
 bill.
 
+.. _accounting/payments/matching-invoices-bills:
+
 Matching invoices and bills with payments
 -----------------------------------------
 
-A blue banner appears when you validate a new invoice/bill and an **outstanding payment** exists for
-this specific customer or vendor. It can easily be matched from the invoice or bill by clicking
-:guilabel:`ADD` under :guilabel:`Outstanding Credits` or :guilabel:`Outstanding Debits`.
+A blue banner appears when validating a new invoice/bill and an **outstanding payment** exists for
+this specific customer or vendor. To match it with the invoice or bill, click :guilabel:`ADD`
+under :guilabel:`Outstanding Credits` or :guilabel:`Outstanding Debits`.
 
 .. image:: payments/add-option.png
    :alt: Shows the ADD option to reconcile an invoice or a bill with a payment.
 
 The invoice or bill is now marked as :guilabel:`In payment` until it is reconciled with its
 corresponding bank statement.
 
-.. _payments/batch-payments:
+.. _accounting/payments/group-payments:
+
+Group payment
+-------------
+
+Group payments allow to combine outgoing and incoming payments from the same partner into a single
```

Comment:
This is a common mistake I see a lot. It should be either:
1. Group payments allow **combining** outgoing and incoming payments
2. Group payments allow **the combination of** outgoing and incoming payments
3. Group payments allow **you** to combine outgoing and incoming payments (of course we want to avoid the unnecessary "you" here, but I wanted to include it so you understand the grammatically correct options.

Reply:
@dade-odoo Thanks for the clarification 🙏